### PR TITLE
espnet2 ngram

### PR DESCRIPTION
**Pull request by qmpzzpmq**

### CODE REVIEW

**Comment by qmpzzpmq** [comment]
Hi, everyone, I am done with this PR. I didn't tuning the decode parameter but still get 4.5%/4.7% WER. It looks fine.

File: egs2/TEMPLATE/asr1/asr.sh [hunk 1]
```diff
@@ -844,7 +856,19 @@ if ! "${skip_train}"; then
...
 
 
+    mkdir -p ${ngram_exp}
     if [ ${stage} -le 9 ] && [ ${stop_stage} -ge 9 ]; then
+        if "${use_ngram}"; then
+            log "Stage 9: Ngram Training: train_set=${data_feats}/lm_train.txt"
+            cut -f 2 -d " " ${data_feats}/lm_train.txt | lmplz -S "20%" --discount_fallback -o ${ngram_num} - >${ngram_exp}/${ngram_num}gram.arpa
+            build_binary -s ${ngram_exp}/${ngram_num}gram.arpa ${ngram_exp}/${ngram_num}gram.bin 
+        else
+            log "Stage 9: Skip ngram stages: use_ngram=${use_ngram}"
```
Comment (ShigekiKarita) [inline egs2/TEMPLATE/asr1/asr.sh:866]: Can you increase this `--stop-stage 9` to 10? https://github.com/espnet/espnet/blob/586eb99492210f706a7c6cfc372a769485471b48/ci/test_integration.sh#L227
@kamo-naoyuki do you know anything else depending on the stage numbers >= 9 in espnet2?

**Review by ShigekiKarita — APPROVED** [review]
Thanks! LGTM overall except for the small request on ci/test_instegration.sh.

**Comment by sw005320** [comment]
Many thanks @qmpzzpmq!

@takaaki-hori, if you review this PR, please continue it here.
We welcome your valuable input!

**Pull request merged**